geopandas/base.py

```diff
@@ -2627,6 +2627,16 @@ def bounds(self):
         0   2.0   1.0   2.0   1.0
         1   0.0   0.0   1.0   1.0
         2   0.0   1.0   1.0   2.0
+
+        You can assign the values to the ``GeoDataFrame`` as:
+
+        >>> gdf["minx"], gdf["miny"], gdf["maxx"],
+        ... gdf["maxy"] = gdf["geometry"].bounds.values.T
```

Comment:
I think it is certainly fine to include this howto as an example here. I am only wondering if this is the best way. Another option could be:

```
gdf = pd.concat([gdf, gdf.bounds], axis=1)
```

Reply:
+1 for using concat instead of accessing an underlying array and transposing it.

Reply:
Agreed thanks. Part of motivation for this PR was to work out the simplest way to do this